eval.py

```diff
@@ -388,6 +389,7 @@ def get_occluded_stats( num_tp_occluded: Number, num_gt_occluded: Number,
       num_tp_occluded : Number of true positives matching occluded ground truth
       num_gt_occluded : Number of occluded ground truth positives in evaluation
       precision       : Overall system precision
+      prefix          : Optional prefix for return result keys (default='')
     Returns
       dict containing statistics with keys 'occluded_recall' and
```

Comment:
The prefixes in this documentation are out of step with the code. I suppose  it should say something like  `keys prefix+'recall' and `... etc.